stack-templates.json

```diff
@@ -1123,7 +1123,7 @@
       "22/tcp"
     ],
     "privileged": true,
-    "registry": "anonymousproxy:8081",
+    "registry": "registry.camunda.cloud/team-cambpm",
```

Comment:
how does Portainer authenticate against Harbor?

Reply:
it is configured here http://portainer.camunda.cloud/#/registries/2

Reply:
Documented here https://github.com/camunda/infra-core/blob/f6601605b6906484f60265ccdd4a42d82858677d/cmd/portainer/setup-infrastructure.sh#L37